packages/providers/fal/image-to-video/fal_image_to_video/models/veo.py

```
@@ -0,0 +1,112 @@
+        }
+
+    def get_model_info(self) -> Dict[str, Any]:
+        """Get Veo 3.1 Fast model information."""
+        return {
+            **MODEL_INFO.get("veo_3_1_fast", {}),
+            "endpoint": self.endpoint,
+            "price_per_second": self.price_per_second,
+            "supports_audio": True
+        }
+
+    def estimate_cost(self, duration, generate_audio: bool = True, **kwargs) -> float:
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)